.editorconfig

```diff
@@ -0,0 +1,36 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+# editorconfig-tools is unable to ignore longs strings or urls
+max_line_length = off
+trim_trailing_whitespace = true
+
+[*.json]
+indent_size = 2
+
+[*.scss]
+indent_size = 2
+
+[*.yml]
+indent_size = 2
+
+# Don't enforce indent size on these types of files
+[*.{md,txt}]
+indent_size = none
+
+# Ignored paths
+[*.{psd,opts}]
+charset = none
+end_of_line = none
+indent_size = none
+indent_style = none
+insert_final_newline = none
+trim_trailing_whitespace = none
+
+[db/seeds/data/libraries.yml]
```

Comment:
I would exclude all seed data here, not just this specific file?

Reply:
Should it be `db/seeds/*` or `db/seeds/data/*`?

Reply:
data :)